Riot/Modules/MatrixKit/Categories/NSString+MatrixKit.swift

```diff
@@ -63,4 +68,14 @@ public extension NSString {
         
         return detectedURL
     }
+
+    /// Returns if the string contains an RTL override character
+    @objc func mxk_containsRTLOverride() -> Bool {
+        return contains(Constants.RTLOverrideChar)
```

Comment:
I'm wondering if the unicode `\u{202E}` can be combined with others into a single character?Maybe this check should drop down to the `unicodeScalars` property of the string just in case (although admittedly that's a world I don't know much about other than the fact there is an `isEmoji` property 🙃).

Reply:
Note: I hadn't noticed this was `NSString` rather than `String`. Possibly makes this comment irrelevant.

Reply:
Yes, seems so :)